test/shared/src/test/scala/monocle/function/AtSpec.scala

```diff
@@ -1,17 +1,35 @@
 package monocle.function
 
 import cats.kernel.Eq
-import monocle.MonocleSuite
+import monocle.{Lens, MonocleSuite}
 import monocle.law.discipline.function.AtTests
 
 import scala.collection.immutable.ListMap
 
 class AtSpec extends MonocleSuite {
   implicit val eqListMap: Eq[ListMap[Int, String]] = Eq.fromUniversalEquals
 
-  implicit def mmapAt[K, V]: At[MMap[K, V], K, Option[V]] = At.fromIso(MMap.toMap)
+  implicit def mmapAt[K, V]: At[MMap[K, V], K, Option[V]] =
+    At.fromIso(MMap.toMap)
 
   checkAll("fromIso", AtTests[MMap[Int, String], Int, Option[String]])
 
   checkAll("ListMap", AtTests[ListMap[Int, String], Int, Option[String]])
+
+  test("atOrElse") {
```

Comment:
Might be nice to have a test for nested maps? Obviously it _should_ just work but given that it's one of the primary motivations for this, it might be nice to test it explicitly

Reply:
Actually, this might be better in `AtExample` if that is considered documentation of how to use it. Usage of `at` for nested maps is definitely not completely obvious at first

Reply:
Make sense. I am thinking to get rid of `example` module and instead use the website. It is probably easier to access.